<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Transfer_Func1D</class>
 <widget class="QWidget" name="Transfer_Func1D">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1131</width>
    <height>599</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>1D Transfer Function</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
     <widget class="QGroupBox" name="hist_groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>161</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Histogram</string>
      </property>
      <widget class="QPushButton" name="Binsize_pushButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>141</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Change Bin Size</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>24</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Bin Size</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="binsize_textEdit">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
      </widget>
      <widget class="QLabel" name="hist_label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Histogram Brightness</string>
       </property>
      </widget>
      <widget class="QSlider" name="hist_brightness_horizontalSlider">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>20</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="TransferFunc_groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>160</y>
        <width>161</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Transfer Function</string>
      </property>
      <widget class="QRadioButton" name="Curve_radioButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Curve</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="StraightLine_radioButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>121</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Straight Line</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="alpha_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>521</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>alpha:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="show_alpha_textEdit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>520</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="isovalue_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>450</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Isovalue:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="show_iso_textEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>481</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>240</y>
        <width>151</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QRadioButton" name="ScalarV_radioButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>111</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Scalar Value</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QComboBox" name="ScalarV_comboBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Frequency</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Moment 2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Moment 3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Moment 4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Skewness</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Kurtosis</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Red to Blue  Rainbow</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Blue to Red  Rainbow</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Rainbow Blended White</string>
       </property>
      </item>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout"/>
   </item>
   <item row="0" column="2">
    <widget class="QSlider" name="hist_scale_verticalSlider">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>551</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>152</width>
       <height>18</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>875</width>
       <height>18</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
